dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -33,6 +33,13 @@ def protected_project?
 
   def report_abuse
     unless protected_project?
+
+      unless verify_recaptcha || !require_captcha?
```

Comment:
Where does `verify_recaptcha` come from?

Reply:
From this `recaptcha` gem we were using in the password reset flow: https://github.com/ambethia/recaptcha#verify_recaptcha